flang/lib/Optimizer/OpenMP/MapsForPrivatizedSymbols.cpp

```diff
@@ -92,7 +103,7 @@ class MapsForPrivatizedSymbolsPass
         /*varPtrPtr=*/Value{},
         /*members=*/SmallVector<Value>{},
         /*member_index=*/mlir::ArrayAttr{},
-        /*bounds=*/ValueRange{},
+        /*bounds=*/boundsOps.empty() ? SmallVector<Value>{} : boundsOps,
```

Comment:
This is unnecessary. An empty `boundsOps` is already an empty `SmallVector<Value>`.